frontend/packages/text-editor/src/TextRow.tsx

```diff
@@ -47,20 +41,19 @@ export const TextRow = ({
   const { t } = useTranslation();
   const [isConfirmDeleteDialogOpen, setIsConfirmDeleteDialogOpen] = useState<boolean>();
 
-  const handleTextIdChange = (event: React.ChangeEvent<HTMLInputElement>): void => {
-    const newTextId = event.currentTarget.value;
+  const handleTextIdChange = (newTextId: string): void => {
+    let error: string;
     if (newTextId !== textId) {
-      if (idExists(newTextId)) {
-        setKeyError('Denne IDen finnes allerede');
-      } else {
-        setKeyError(validateTextId(newTextId));
-      }
-      setTextIdValue(newTextId);
+      error = idExists(newTextId) ? 'Denne IDen finnes allerede' : validateTextId(newTextId);
+    } else {
+      error = '';
     }
+    setKeyError(error);
+    setTextIdValue(newTextId);
   };
```

Comment:
Github did not allow me to add suggestion on unchanged lines. 🙈 Howerver, what do you think about refactoring the code by splitting the `handleTextChange `into multiple parts? 

```typescript
const handleTextIdChange = (newTextId: string): void => {
  const errorMessage= validateNewTextId(newTextId);
  
  setKeyError(errorMessage || '');
  setTextIdValue(newTextId);
};

const validateNewTextId = (newTextId: string): string | null => {
  if (newTextId === textId) {
    return null;
  }

  if (idExists(newTextId)) {
    return 'Denne IDen finnes allerede';
  }

  return validateTextId(newTextId);
};
```

If there are no error messages, we return null. This makes it clear that when we set an error and there are no validation errors, we assign an empty string. If we decide to proceed with this change, it is important to modify the implementation of validateTextId from the utility (utils) to also return null if no errors are found.

Reply:
We should also add the text "Denne IDen finnes allerede" to our languages  files.